### PR TITLE
[TASK] Drop support for PHP 7.2 and 7.3

### LINKED ISSUE #577: Drop the PHP 7.2/7.3 workaround for MySQL from the CI scripts
… once we have dropped support for PHP 7.2 and 7.3.

### PR DESCRIPTION
**Pull request by oliverklee**
Also sync the test matrices of the `predefined` CI jobs with those from the Composer-script-driven CI jobs.

This change does not add any 7.4-only PHP language features.

Fixes #577

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Deprecated
 
 ### Removed
+- Drop support for PHP 7.2 and 7.3 (#581)
```
Comment (brotkrueml) [inline CHANGELOG.md:15]:
```suggestion
- Support for PHP 7.2 and 7.3 (#581)
```

It is read: "Removed support for ...", so I suggest to avoid "Drop".

**Review by brotkrueml — COMMENTED** [review]
I assume, phpunit and maybe other dependencies are updated in a follow-up PR?

**Comment by oliverklee** [comment]
> I assume, phpunit and maybe other dependencies are updated in a follow-up PR?

Correct. I'd like to keep the PRs small and focused ("only one topic per PR").

File: CHANGELOG.md (hunk 1 above)
Comment (oliverklee) [reply CHANGELOG.md:15]: It'd prefer the changelog lines to always be worded like good commit message subjects. i.e., starting with a imperative verb like explained here: https://cbea.ms/git-commit/

And the commit message entries should make sense on its own, i.e., they should work without having to add the section heading. (This also is how we have tried to write the changelog entries so far.)

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**